pkg/apis/pravega/v1alpha1/pravega.go

```diff
@@ -332,10 +332,11 @@ type FileSystemSpec struct {
 
 // ECSSpec contains the connection details to a Dell EMC ECS system
 type ECSSpec struct {
-	ConfigUri   string `json:"configUri"`
-	Bucket      string `json:"bucket"`
-	Prefix      string `json:"prefix"`
-	Credentials string `json:"credentials"`
+	ConfigUri    string `json:"configUri"`
+	Bucket       string `json:"bucket"`
+	Prefix       string `json:"prefix"`
+	Credentials  string `json:"credentials"`
```

Comment:
The field "Credentials" is perhaps expected to contain the name of the k8s secret that has Credentials for tier2 ??? 
https://github.com/pravega/pravega-operator/blob/7665c42d9ac85cf4d8def42a4f369b74c8918258/pkg/controller/pravega/pravega_segmentstore.go#L299

Reply:
Yes, both ECS credentials and certificates are k8s secret references.